rust/exchange_rate/src/main.rs

```diff
@@ -238,13 +250,15 @@ async fn get_rate(job: Timestamp) {
     }
 }
 
-fn keep_bucket_start_time_and_closing_price(body: &[u8]) -> Vec<u8> {
+fn keep_bucket_start_time_and_closing_price(body: &[u8], context: &Vec<u8>) -> Vec<u8> {
```

Comment:
usually the more convension and the more consistent way of passing the content is just to use a slice - &[u8]